src/views/prize-earned/prize-earned.scss

```diff
@@ -0,0 +1,46 @@
+h1 {
+  text-align: center;
+  margin-top: 72px;
+}
+
+.trophy-banner {
+  margin-top: 141px;
+  position: relative;
+}
+
+.trophy {
+  position: relative;
+  z-index: 1;
+  margin-top: -250px;
+  left: 50%;
+  margin-left: -187.5px; // half ot the image width
+}
+
+.top-line {
+  margin-top: -70px;
+}
+
+.parent {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
```

Comment:
you shouldn't need `margin: 0 auto;` if you're using `justify-content: center;`

Reply:
removed